### PR TITLE
Add basic docs

### PR DESCRIPTION
**Pull request by FebinKa**
Adding README.md as documentation using Sphinx to address issue #106

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = "euro-calliope-workflow"
+copyright = "2021, calliope-project authors"
+author = "calliope-project authors"
+
+# The full version, including alpha/beta/rc tags
+release = "1.1"
```
Comment (sjpfenninger) [inline docs/source/conf.py:26]: Should the version be pulled from somewhere else, e.g. at the top level of the repo?

**Review by sjpfenninger — CHANGES_REQUESTED** [review]
Looks good. I have only one comment regarding setting the version number, and a question:

Should the dependencies be in the environment.yml file (I added them in there for now) or a separate docs file? If we later build DAG graphics for the docs I guess we need all requirements in one place to build the docs e.g. on RTD.

I also made two minor changes:

* increased header level in markdown file as prep for splitting across top-level headings
* changed to RTD sphinx theme (cleaner than alabaster I think but happy to revert if you disagree)

**Review by sjpfenninger — APPROVED** [review]

**Comment by timtroendle** [comment]
@FebinKa @sjpfenninger For my understanding: you copied the content from README.md to the new doc. As with any copy, it's only a matter of time until the two will get out of sync. How do you think we should handle that?

**Comment by FebinKa** [comment]
It's not exactly the same as @sjpfenninger changed the header levels and the file was formatted to include the sources before the license. The goal would be to address #103 through the Sphinx documentation and using the copy allows to paste the new inputs at the appropriate places. However, it would also be possible to [directly link](https://stackoverflow.com/a/17217041/15452089) to the readme instead by creating `readme_link.rst` inside `docs/source` which contains ` include:: ../../README.md` and calling that instead of `overview` in `index.md`.

**Comment by timtroendle** [comment]
Thanks for the clarification. I am really just worried that we are now maintaining two things, and we should have a strategy moving forward how to handle that.

**Comment by brynpickering** [comment]
@FebinKa I like the suggestion to point to the README directly to fill in the 'overview' information, instead of the copy-paste job we did. Perhaps you could implement that? In fact, the index could just show that, then we don't have to worry about having moved some but not all things from the readme to the index. The sub-pages can be reserved for more details on the workflow.

**Pull request merged**